src/test/resources/regressions/examples/tutorial-examples/external-interface/counterImpl/counterImpl.gobra

```diff
@@ -6,17 +6,18 @@ package counterImpl
 // implementation
 type counter struct{ f, max int }
 
+decreases
```

Comment:
in some examples, we put the `decreases` clause before the pre/posts, in others we put it after. I would rather use a single style that we consider idiomatic throughout the examples (with a preference for putting termination measures after pre/posts)

Reply:
I'm used to put it first as it's usually shorter but I'm fine with settling for putting it last